trimmer_decorator.ru

```diff
@@ -0,0 +1,8 @@
+require_relative 'decorator'
+class TrimmerDecorator < Decorator
+  def correct_name
+    return unless @nameable.correct_name.length > 10
+
+    correct_name.strip
```

Comment:
- [ ] Kindly make sure that your `TrimmerDecorator` method works correctly, if the characters  is `longer  than 10 it should  be trimmed`   

```suggestion
   @nameable.correct_name.length > 10 ? @nameable.correct_name[0..9] : @nameable.correct_name
```